### PR TITLE
TCVP-1751

### PR DESCRIPTION
**Pull request by Sharkww**
# Description

This PR includes the following proposed change(s):

- TCVP-1751
- UI/UX fix for pages
- updated enum
- updated .env-template

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+      <div class="col-md-6">
+        <mat-form-field class="w-100">
+          <mat-label>Surname</mat-label>
+          <input matInput formControlName="disputant_surname" />
+          <mat-error>{{ "error.required" | translate }}</mat-error>
+        </mat-form-field>
+      </div>
+      <div class="col-md-6">
+        <mat-form-field class="w-100">
+          <mat-label>Given names</mat-label>
```
Comment (lfdgov) [inline src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html:13]: Please add a hint "Use legal surname"

File: src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
+<form [formGroup]="form" novalidate>
+  <section class="mb-2">
+    <div class="row">
+      <div class="col-md-6">
+        <mat-form-field class="w-100">
+          <mat-label>Surname</mat-label>
+          <input matInput formControlName="disputant_surname" />
+          <mat-error>{{ "error.required" | translate }}</mat-error>
+        </mat-form-field>
+      </div>
```
Comment (lfdgov) [inline src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html:10]: In this and the regular dispute stepper, please change the mat-lables for surname, given name to:
Legal Surname
Legal Given Name(s)

File: src/frontend/citizen-portal/src/app/shared/components/resolution-header/resolution-header.component.html [hunk 3]
```diff
@@ -1,22 +1,17 @@
...
-    <div class="col col-lg-3" style="text-align: end;">
+    <div fxFlex style="text-align: end;">
       <small class="me-2 text-nowrap">Violation time:</small>
-      <strong class="text-nowrap">{{ ticket.issued_date | date: "HH:mm" }}</strong>
-      <strong class="d-md-inline d-lg-none">{{
-        ticket.issued_date | date:" yyyy-MM-dd"
-      }}</strong>
+      <b class="text-nowrap">{{ time }}</b>
+      <b *ngIf="date" fxHide.gt-md="true" fxShow.lt-lg="true"> {{ date }}</b>
     </div>
```
Comment (lfdgov) [inline src/frontend/citizen-portal/src/app/shared/components/resolution-header/resolution-header.component.html:11]: did you verify the date and time formatting with the business?

**Review by lfdgov — CHANGES_REQUESTED** [review]

File: src/frontend/citizen-portal/src/app/shared/components/resolution-header/resolution-header.component.html (hunk 3 above)
Comment (Sharkww) [reply src/frontend/citizen-portal/src/app/shared/components/resolution-header/resolution-header.component.html:11]: moved to ts

File: src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html (hunk 2 above)
Comment (Sharkww) [reply src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html:10]: done

**Review by lfdgov — APPROVED** [review]

**Pull request merged**